Formula/micromamba.rb

```diff
@@ -1,11 +1,17 @@
 class Micromamba < Formula
   desc "Fast Cross-Platform Package Manager"
   homepage "https://github.com/mamba-org/mamba"
-  url "https://github.com/mamba-org/mamba/archive/refs/tags/2023.02.03.tar.gz"
-  version "1.3.0"
-  sha256 "4c794246dd6008caebebef8c6266e6ba3f9ff5ce48908c830e448811587e9921"
+  url "https://github.com/mamba-org/mamba/archive/refs/tags/micromamba-1.3.1.tar.gz"
+  sha256 "af66b79bc6b9673209e92b6ce6e52a21a79c55e86960a9d5bb10691cf2b4e327"
   license "BSD-3-Clause"
 
+  livecheck do
```

Comment:
I can't leave a proper suggestion from the mobile website but this `strategy` block is unnecessary and this should simply use `regex(/^micromamba[._-]v?(\d+(?:\.\d+)+)$/i)` instead.

Reply:
Thanks!